### DISCUSSION TITLE
How to plan a route: Kuviger web ?, Cruiser Desktop ?, BRouter web ?

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#295] [Q&A] [answered]
These last years I used "https://kurviger.de" to plan the routes, exported to ".kurviger" and imported the file with "Kurviger" and then with "Kurviger 2"
After finishing the development of "Kurviger 2" I tried ""Kurviger 3" and "Cruiser" and obviously I prefer "Cruiser" ... probably like many people.

Now I'm in a moment to see how I do the plannings: "https://kurviger.de" works great for me but I'm looking at an alternative and so I was thinking with "Cruiser Desktop" where I created a route, exported as ". kurviger" and then imported from "Cruiser" and it seems to work perfectly.
I think BRouter doesn't allow exporting to ".kurviger"

What I'm interested in is preparing the route from the PC and importing it with Cruiser App, with all the advantages of the ".kurviger" format.

Thanks !

**Comment by 0709wiwiwi** [comment]
> I think BRouter doesn't allow exporting to ".kurviger"

No but export as a gpx navigation track offers you the same navigation experience.
Cruiser app (+desktop)  converts <> into the file format you do prefer to use.

Current Brouter web gpx navigation track export set:  
https://github.com/devemux86/cruiser/discussions/250#discussioncomment-6508754

**Comment by devemux86** [comment]
@ramondevic very nice question!

On desktop it is best to use the [BRouter website](https://brouter.de/brouter-web/).
- Here is Cruiser's driving profile and user instructions:
https://github.com/devemux86/cruiser/discussions/272

- For export we can use the GPX navigation format (**offline** and contains everything):
https://github.com/devemux86/cruiser/discussions/250#discussioncomment-6508754

We hope in the future that BRouter website becomes easier to load profiles and export GPX navigation files.

cc @afischerdev

**Reply by ramondevic** [reply]
I am doing several tests to see if I understand it and it works for me.

When I talk about ".kurviger" it's because when importing the file in this format you already have all the navigation data and nothing is recalculated when importing, I'm also interested in street names and speed limits which seem to only be seen with the ".kurviget" format and with Graphopper.
With Cruiser I use Graphopper for when I manually create a short route to get that data as well.

Little by little we will get it ;)

**Reply by devemux86** [reply]
> when importing the file in this format you already have all the navigation data and nothing is recalculated when importing

This is also possible when export / import in GPX navigation format.
It contains all routing + navigation data and can be imported **offline**.

> I'm also interested in street names and speed limits

These depend on the router, especially the speed limits. :slightly_smiling_face: 
- https://github.com/devemux86/cruiser/discussions/135

**Comment by DolomitesTom** [comment]
I plan many day trips with cruiser Android, works fine for me. Longer trips I plan with Kurviger Web, fill automatically or manually with some shaping points. Import in  Cruiser in gpx or kurviger format. When editing in Cruiser I export in Kurviger format, because then I can later import again into the Kurviger Cloud 1:1 without any change.

**Comment by Cer-o** [comment]
At the moment the solution I have adopted is to continue supporting both projects (Cruiser and Kurviger) and take advantage of the best of both.
Planning with Kurviger web or android and navigation with Cruiser.

**Comment by WolfgangZ42** [comment, marked as answer]
Three different approaches:

- For trips by car, I use Cruiser alone with BRouter (offline) and thus also benefit from the NoGo area for the Diesel ban in Munich. 
- For offroad-tracks (ACT, TET, etc.) I also use Cruiser and the profile Mountainbike to create a navigatable route. 
- "Quite normal" motorcycle tours with several participants I create with kurviger.de because of the simple and fast definition of a tour and the ingenious routing of the Kurviger server. 

All motorcycle tours I drive consequently without recalculation - and I extend these routes with enough shaping points to limit the algorithms of Garmin/TomTom devices and get an identical route (mostly). This extension can be done manually, via export with "additional waypoints" in kurviger.de or as described in my workflow: [Workflow for Kurviger.pdf](https://github.com/devemux86/cruiser/files/12431056/Workflow.for.Kurviger.pdf)

Btw: bicycle and hiking routes I plan with Komoot or BRouter web and navigate with Komoot (first tests with Cruiser were quite positiv!).

**Comment by airald** [comment]
I use another plattform, furkot.com 
I use for all my routes since 5 years ago. You can plan long travels easily. As it uses OSM and I usually place a lot of waypoints, I usually don't have problems when I load GPX to Cruiser.

A lot of options available, and user inteface is clean and easy to use. Is not free but you can try all features except GPX export. 10 or 12 € yearly. Even you can share the route with others to view or edit. Control the daily milleage to plan night stops and also the fuel range or search POIs from different sources.
